### ISSUE TITLE
Add more than 1 user + Read-Only public view

### ISSUE THREAD

**Issue by snypelife** [cryptoGlance/cryptoGlance-web-app#16]
- Enable the use of more than one user
-  Possibly allow other users to be READ ONLY
-  Properly handle the method to reset passwords/accounts if one already exists, but the 2nd+ account is created or needs to be reset

**Comment by missura** [comment]
yes. more accounts, or ability to disable login screen, and i can controll with .htaccess
